pkg/db/tx_test.go

```diff
@@ -91,4 +90,18 @@ func TestGet(t *testing.T) {
 			t.Errorf("got %s wanted not found", err)
 		}
 	})
+
+	t.Run("failure unique", func(t *testing.T) {
+		_, err := d.Transact(ctx, func(tx db.Tx) (interface{}, error) {
+			var r R
+			_ = tx.Get(&r, `CREATE TABLE test_table (ID int NOT NULL UNIQUE)`)
```

Comment:
would call the table "test_failure_unique" just in case somebody else testing the database here and will come up with the same name

Reply:
Do you think we should delete the table at the end as well?